assets/src/design-system/components/button/button.js

```diff
@@ -126,10 +126,39 @@ const tertiaryColors = ({ theme }) => css`
   }
 `;
 
+const quaternaryColors = ({ theme }) => css`
+  background-color: ${theme.colors.interactiveBg.quaternaryNormal};
+  border: 1px solid ${theme.colors.border.defaultNormal};
+
+  &:hover {
+    border-color: ${theme.colors.border.quaternaryHover};
+    background-color: ${theme.colors.interactiveBg.quaternaryNormal};
+  }
+
+  &:focus {
+    box-shadow: none;
+    border-color: ${theme.colors.border.quaternaryHover};
+    background-color: ${theme.colors.interactiveBg.quaternaryNormal};
+  }
+
+  &:active {
+    border-color: ${theme.colors.border.quaternaryActive};
+    background-color: ${theme.colors.interactiveBg.quaternaryNormal};
+  }
+
+  ${themeHelpers.focusableOutlineCSS};
+
+  &:disabled {
+    border-color: ${theme.colors.border.disable};
+    background-color: ${theme.colors.quaternaryNormal};
+  }
```

Comment:
Does the background-color need to be specified since it's always the same? It's also defined on line 130 so I figured this is probably not necessary?
```suggestion
  &:hover {
    border-color: ${theme.colors.border.quaternaryHover};
  }

  &:focus {
    box-shadow: none;
    border-color: ${theme.colors.border.quaternaryHover};
  }

  &:active {
    border-color: ${theme.colors.border.quaternaryActive};
  }

  ${themeHelpers.focusableOutlineCSS};

  &:disabled {
    border-color: ${theme.colors.border.disable};
  }
```

Reply:
So i got a little greedy, background color is needed to override the default button base active and disabled background colors. II was able to remove it on hover and focus though.